contents/docs/surveys/troubleshooting.mdx

```diff
@@ -53,3 +53,14 @@ You can call `getActiveMatchingSurveys` in your browser console to see which sur
 ```js
 posthog.getActiveMatchingSurveys(surveys => console.log(surveys))
 ```
+
+### Why am I getting a feature flags error on my survey?
+
+```Update survey failed: Cohort with behavioral filters cannot be used in feature flags.
+```
+
+Surveys are powered by feature flags for user property targeting. Currently we do not support behavioral cohorts in flags which is why you might see an error if setting a user property that contains a behavioral cohort in your survey.
+
+#### Will I be double charged for feature flags if I use surveys?
+
+No, you will not.
```

Comment:
Do we have a link to the pricing explanation we can add here?

Reply:
Pricing explanation for surveys? or flags? or both?

Reply:
I'll add more details if I continue to get confusion from users!